webant/api/blueprint_api.py

```diff
@@ -1,34 +1,10 @@
-import json
-import tempfile
-import os
-
-from flask import Blueprint, current_app, jsonify, request, url_for
-from werkzeug import secure_filename
-
-from archivant.archivant import Archivant
-from archivant.exceptions import NotFoundException
-from webant.util import send_attachment_file
-
-
-class ApiError(Exception):
-    def __init__(self, message, http_code, err_code=None, details=None):
-        Exception.__init__(self, http_code, err_code, message, details)
-        self.http_code = http_code
-        self.err_code = err_code
-        self.message = message
-        self.details = details
-
-    def __str__(self):
-        return "http_code: {}, err_code: {}, message: '{}', details: '{}'".format(self.http_code, self.err_code, self.message, self.details)
-
-
-def make_success_response(message, http_code=200):
-    response = jsonify({'code': http_code, 'message': message})
-    response.status_code = http_code
-    return response
-
+from flask import Blueprint, current_app, jsonify
+from webant.util import add_routes
+from archivant_api import routes
```

Comment:
this part is slightly counterintuitive: all this pull request is created so to make multiple modules adding routes to a single blueprint easier. But if you do `from foo import routes` you can only do it one time.
So probably
```python
import archivant_api

add_routes(api, archivant_api.routes)
```

would be easier